src/main/java/UseCase/RegisterArtistInteractor.java

```diff
@@ -0,0 +1,41 @@
+package UseCase;
+
+import Database.userAccessInterface;
+import Database.userDsData;
+import Entities.User;
+import Entities.userFactory;
+import InputBoundary.RegisterArtistInputBoundary;
+import InputData.RegisterArtistInputData;
+import OutputBoundary.loginOutputBoundary;
+
+public class RegisterArtistInteractor implements RegisterArtistInputBoundary{
+    private final loginOutputBoundary registerPresenter;
+    private final userFactory userFactory;
+
+    userAccessInterface allUsers;
+
+    public RegisterArtistInteractor(loginOutputBoundary registerPresenter, userAccessInterface allUsers,
+                                    userFactory userFactory){
+        this.allUsers = allUsers;
+        this.registerPresenter =registerPresenter;
+        this.userFactory = userFactory;
```

Comment:
I'm not certain if userFactory would be passed in like this, actually. @sarxhlwalker I know you removed spaceFactory, but are you more familiar with this?

Reply:
I'm not super familiar, but I think it would make more sense to construct a new userFactory within the interactor (as technically userFactory is an entity, so I don't believe you should be creating a new instance of an entity outside of the red layer). it doesn't take any construction parameters, so it wouldn't really change anything, I don't think.

Reply:
userFactory is now constructed in the constructer, thanks for the comment